demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -90,6 +93,17 @@ def run_validations(self) -> int:
                         in filtered_content_objects_for_validator
                     ]
                 try:
+                    # check if any of the validators error code appears in ALWAYS_RUN_ON_ERROR_CODE
+                    if any(
+                        error_code in ALWAYS_RUN_ON_ERROR_CODE
+                        for error_code in {
+                            validator.error_code for validator in self.validators
+                        }
+                    ):
+                        validation_results = self.filter_validation_results(
+                            validation_results
+                        )
+
```

Comment:
We are running in the scope of the specific validation, why do we need to check for every error code?

Reply:
This is a change to https://github.com/demisto/demisto-sdk/pull/4983#discussion_r2179346526, where the idea was to first check if any of the validators are in ALWAYS_RUN_ON_ERROR_CODE before filtering the results.
This is due to the uniqe situation of GR107 and any future validations that need to run first, and filter its results later.

Reply:
Fixed.